person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Again here, good job adding the rental method as requested, however, kindly add it in such a way that you create a `new Rental object` and establish its association with a `Book` and a `Person` as done below. The method you are currently using is for when you alraedy have a `Rental` but since here we are creating a new `Rental` we should use the following method 

e.g
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```